actionpack/test/controller/live_stream_test.rb

```diff
@@ -373,6 +373,23 @@ def test_send_stream
       assert_match "my.csv", @response.headers["Content-Disposition"]
     end
 
+    def test_send_stream_instrumentation
+      payload = nil
+
+      subscriber = proc do |*args|
+        event = ActiveSupport::Notifications::Event.new(*args)
+        payload = event.payload
+      end
```

Comment:
```suggestion
      subscriber = proc { |event| payload = event.payload }
```

Following https://github.com/rails/rails/pull/49433.